examples/arm/run.sh

```diff
@@ -217,6 +226,17 @@ function run_fvp() {
             -a "${elf}"                                         \
             --timelimit 120 || true # seconds
         echo "[${FUNCNAME[0]} Simulation complete, $?"
+    elif [[ ${target} == *"ethos-u85"*  ]]; then
+	${fvp_model}                                            \
+	    -C mps4_board.subsystem.cpu0.CFGITCMSZ=11           \
+	    -C mps4_board.subsystem.ethosu.num_macs=128         \
```

Comment:
This has to be aligned with the compile spec used AoT right?

Reply:
It's aligned with the default spec used for now e.g. ethos-u85-128 we will probably add a change to use 
-C mps4_board.subsystem.ethosu.num_macs=${num_macs} 
soon for U85 also to make it more generic but we should probably test it some more before.

Reply:
changed it to num_macs to match the Ethos-U55 pattern. 

We need to do a followup to CI the non-default target, but as it adds a lot of cycles to the testing, we should revise the e2e flow to not run the heavyweight models on most configs.